app/src/main/res/layout/dialog_critical_app_update.xml

```diff
@@ -40,7 +40,7 @@
     android:layout_width="wrap_content"
     android:layout_height="wrap_content"
     android:layout_marginTop="@dimen/spacing_16"
-    android:text="@string/critical_update_required_dialog_error_warning"
+    android:text="@string/critical_security_update_required_reason"
```

Comment:
Since we are deciding which text to show during runtime. Should we remove the default text and use `tools:text` to see in preview?